lib_version.py

```diff
@@ -0,0 +1,25 @@
+# BuildFox ninja generator
+
+import re
+
+MAJOR = 1
+MINOR = 0
+VERSION = "%d.%d" % (MAJOR, MINOR)
+
+# Simple major.minor matcher
+re_version = re.compile(r"^(\d+)\.(\d+)$")
+
+def parse(version):
```

Comment:
I think we should "complicate" stuff as it's connected with user input/output and good error handling is a must.

Also, regex solution is not much longer than `versiontuple`, but it's more fail-proof :)